napari/layers/_tests/test_layer_actions.py

```diff
@@ -213,6 +219,15 @@ def test_convert_layer(layer, type_):
     assert ll[0]._type_string == type_
     assert np.array_equal(ll[0].scale, original_scale)
 
+    if (
+        type_ == "labels"
+        and isinstance(input, Image)
+        and np.issubdtype(input.data.dtype, np.integer)
+    ):
+        assert (
+            input.data is ll[0].data
+        )  # check array data not copied unnecessarily
```

Comment:
Something is wrong with this test as assert is not covered. Please fix.

Reply:
@Czaki it's `input`, which should be `layer`. I've pushed a fix.